c2pa/claim-examples/claim_create.json

```
@@ -0,0 +1,98 @@
+{
+    "vendor": "starlinglab",
+    "recorder": "Starling Capture",
```

1. `"recorder"` should be `"claim_generator"`, and the value should be `"Starling Capture using Numbers Protocol"`.
2. Missing some [necessary fields](https://c2pa.org/public-draft/C2PA_Specification.html#_syntax) including:
    1. `dc:format`, ex: `image/jpeg`
    2. `instanceID`, ex: `xmp:fakeid:4124fae1-1da7-4a3f-95c8-d8ae071bd048`
    3. `signature`, ex: `self#jumbf=c2pa/urn:uuid:F9168C5E-CEB2-4faa-B6BF-329BF39FA1E4/c2pa.signature`
    4. (optionan, but I suggest to include to show more info on CAI's inspect website): `dc:title`: ex: filename

---

Ahhhh it seems those keys are transformed when they pass thru `claim_tool`, so this is actually not valid C2PA, but input format for `claim_tool`. Noted.

Here is an example generated with `claim_tool` (note that some fields are outdated here):
```
{
  "root": "starlinglab:urn:uuid:618c30c1-1a53-42f9-b1d7-ae2987ea7967",
  "manifests": {
    "starlinglab:urn:uuid:618c30c1-1a53-42f9-b1d7-ae2987ea7967": {
      "claim": {
        "alg": "sha256",
        "assertions": [
          {
            "alg": "sha256",
            "hash": "<omitted>",
            "url": "self#jumbf=c2pa/starlinglab:urn:uuid:618c30c1-1a53-42f9-b1d7-ae2987ea7967/c2pa.assertions/c2pa.thumbnail.claim.jpeg"
          },
          {
            "alg": "sha256",
            "hash": "<omitted>",
            "url": "self#jumbf=c2pa/starlinglab:urn:uuid:618c30c1-1a53-42f9-b1d7-ae2987ea7967/c2pa.assertions/adobe.beta"
          },
          {
            "alg": "sha256",
            "hash": "<omitted>",
            "url": "self#jumbf=c2pa/starlinglab:urn:uuid:618c30c1-1a53-42f9-b1d7-ae2987ea7967/c2pa.assertions/stds.schema-org.CreativeWork"
          },
          {
            "alg": "sha256",
            "hash": "<omitted>",
            "url": "self#jumbf=c2pa/starlinglab:urn:uuid:618c30c1-1a53-42f9-b1d7-ae2987ea7967/c2pa.assertions/c2pa.actions"
          },
          {
            "alg": "sha256",
            "hash": "<omitted>",
            "url": "self#jumbf=c2pa/starlinglab:urn:uuid:618c30c1-1a53-42f9-b1d7-ae2987ea7967/c2pa.assertions/stds.exif"
          },
          {
            "alg": "sha256",
            "hash": "<omitted>",
            "url": "self#jumbf=c2pa/starlinglab:urn:uuid:618c30c1-1a53-42f9-b1d7-ae2987ea7967/c2pa.assertions/stds.iptc.photo-metadata"
          },
          {
            "alg": "sha256",
            "hash": "<omitted>",
            "url": "self#jumbf=c2pa/starlinglab:urn:uuid:618c30c1-1a53-42f9-b1d7-ae2987ea7967/c2pa.assertions/org.starlinglab.integrity"
          },
          {
            "alg": "sha256",
            "hash": "<omitted>",
            "url": "self#jumbf=c2pa/starlinglab:urn:uuid:618c30c1-1a53-42f9-b1d7-ae2987ea7967/c2pa.assertions/c2pa.hash.data"
          }
        ],
        "claim_generator": "Starling Capture",
        "dc:format": "image/jpeg",
        "dc:title": "ad6f62fa-cb24-4421-9c44-c3159601994e.jpg",
        "instanceID": "xmp:iid:fcf291e1-6dec-46ad-8cf7-4d026d3d09b6",
        "signature": "self#jumbf=c2pa/starlinglab:urn:uuid:618c30c1-1a53-42f9-b1d7-ae2987ea7967/c2pa.signature"
      },
      "assertion_store": {
        "c2pa.hash.data": {
          "alg": "sha256",
          "exclusions": [
            {
              "length": 111496,
              "start": 20
            }
          ],
          "hash": "<omitted>",
          "name": "jumbf manifest",
          "pad": "<omitted>"
        },
        "stds.exif": {
          "@context": {
            "exif": "http://ns.adobe.com/exif/1.0/"
          },
          "exif:GPSLatitude": "43/1 38/1 7546907075566109/1125899906842624",
          "exif:GPSLatitudeRef": "N",
          "exif:GPSLongitude": "79/1 24/1 6721622443850465/1125899906842624",
          "exif:GPSLongitudeRef": "W",
          "exif:GPSTimeStamp": "2021:11:22 23:18:27 +0000"
        },
        "c2pa.thumbnail.claim.jpeg": "<omitted> len = 100571",
        "adobe.beta": {
          "version": "0.7.4"
        },
        "stds.iptc.photo-metadata": {
          "@context": {
            "Iptc4xmpExt": "http://iptc.org/std/Iptc4xmpExt/2008-02-29/",
            "dc": "http://purl.org/dc/elements/1.1/"
          },
          "Iptc4xmpExt:LocationCreated": {},
          "dc:creator": [
            "Benedict Lau"
          ],
          "dc:rights": "Copyright (C) 2021 Hypha Worker Co-operative. All Rights Reserved."
        },
        "c2pa.actions": {
          "actions": [
            {
              "action": "c2pa.created",
              "when": "{{created_timestamp}}"
            }
          ]
        },
        "org.starlinglab.integrity": {
          "starling:Identifier": "{{starling_identifier}}",
          "starling:Signatures": [
            {
              "starling:AuthenticatedPublicData": {
                "starling:AssetCreatedTimestamp": "{{starling_signature_authenticated_public_data_asset_created_timestamp}}",
                "starling:AssetHash": "{{starling_signature_authenticated_public_data_asset_hash}}",
                "starling:AssetMimeType": "{{starling_signature_authenticated_public_data_asset_mime_type}}"
              },
              "starling:Provider": "{{starling_signature_provider}}",
              "starling:PublicKey": "{{starling_signature_public_key}}",
              "starling:SignedMessage": "{{starling_signature_signed_message}}",
              "starling:SignedMessageDescription": "{{starling_signature_signed_message_description}}"
            }
          ]
        },
        "stds.schema-org.CreativeWork": {
          "@context": "https://schema.org",
          "@type": "CreativeWork",
          "author": [
            {
              "@type": "Person",
              "credential": [],
              "identifier": "https://hypha.coop",
              "name": "Benedict Lau"
            }
          ]
        }
      },
      "signature": "<omitted> len = 7019"
    }
  }
}
```

---

Yup this one I was going to add today, thanks!

```suggestion
    "recorder": "Starling Capture by Numbers Protocol",
```

---

At some point I should change `recorder` to be `claim_generator` to match the  updated c2pa spec.